### DISCUSSION TITLE
Use Docker deployment for recommendations

### DISCUSSION THREAD

**Discussion by Ran-Xing** [coder/code-server#4139] [Show and tell]
Hello, Some time ago, I deployed a docker container for code in a week
Github: https://github.com/XRSec/Code-Server-Update
Dockerfile : https://github.com/XRSec/Code-Server-Update/blob/main/Dockerfile
Docker: https://hub.docker.com/r/xrsec/vscode

I hope you can have a look sometime. Tks

**Comment by jsjoeio** [comment]
This is awesome! So basically it's a Docker environment with code-server + PHP tools? Super cool!

cc @bpmct

**Reply by bpmct** [reply]
Thanks for sharing! Cool to see how you handled the PHP environment and code-server auth in the Dockerfile! Where are you hosting it?

**Reply by Ran-Xing** [reply]
> Thanks for sharing! Cool to see how you handled the PHP environment and code-server auth in the Dockerfile! Where are you hosting it?

Docker container address： https://hub.docker.com/r/xrsec/vscode

But I met a problem. I used to use php5.6 to successfully debug, but I forgot how to change the configuration file. Now php7.4 can debug normally
